docs/updates/migrate-v3.md

```diff
@@ -126,3 +126,15 @@ The following processors have been removed:
 ^2^ Enabled for projects defining a [component preload configuration](../pages/Configuration.md#component-preload-generation)  
 ^3^ Enabled in `self-contained` build, which disables `generateComponentPreload` and `generateLibraryPreload`  
 ^4^ Enabled for projects defining a [bundle configuration](../pages/Configuration.md#custom-bundling)  
+
+## Removal of Standard Middleware
+
+The following middlewares have been removed from the [standard middlewares list](../../pages/Server/#standard-middleware):
```

Comment:
The following middleware has been removed from the [standard middleware list](../../pages/Server/#standard-middleware):

Reply:
Fixed in https://github.com/SAP/ui5-tooling/pull/735